spline-shadow/rest-gateway/src/main/scala/za/co/absa/spline/gateway/rest/AppConfig.scala

```diff
@@ -16,9 +16,45 @@
 
 package za.co.absa.spline.gateway.rest
 
-import org.springframework.context.annotation.{Configuration, Import}
+import java.util.Arrays.asList
+
+import org.apache.commons.configuration.{CompositeConfiguration, EnvironmentConfiguration, SystemConfiguration}
+import org.springframework.context.annotation.{Bean, Configuration, Import}
+import org.springframework.web.filter.CorsFilter
+import org.springframework.web.cors.CorsConfiguration
+import org.springframework.web.cors.UrlBasedCorsConfigurationSource
+import za.co.absa.spline.common.config.ConfTyped
+import za.co.absa.spline.gateway.rest.CorsConfig.CorsFilterConf
 
 @Configuration
 @Import(Array(classOf[ArangoRepoConfig]))
-class AppConfig
+class AppConfig{
+
+
+
+  val source: UrlBasedCorsConfigurationSource  = new UrlBasedCorsConfigurationSource()
```

Comment:
Lines 35-44 should move to  the `springFilterPoxy` method because the `AppConfig` is supposed to contain different loosely related bean definitions. So there should only be `@Bean def .....` on the class members level.